standard/template/generator-template/component/{name_pc}/{name_pc}.styles.tsx.sg

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components';
```

Comment:
Should the style file have the `tsx` extension, even though there is no react? Or was this on purpose to be consistent (or just in case)?

Reply:
1. For consistency.
2. So we can use PascalCase for styled components name and our eslint rule won't complain about it.
3. Just in case, you never know when you might need to add some JSX.

Reply:
I didn't have any problems with regular `.ts` files; Also 3 is not really valid anymore as in the latest react you don't have to import react anymore to use JSX. As it will be handled by babel see: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
re 3) It's more related to your IDE, so it can differentiate between normal JS, or also JSX parsing.